server/game_service.py

```diff
@@ -145,18 +148,32 @@ def create_game(
         }.get(game_mode, Game)
         game = game_class(**args)
 
-        self.games[game_id] = game
+        self._games[game_id] = game
 
         game.visibility = visibility
         game.password = password
 
         self.mark_dirty(game)
         return game
 
+    def update_active_game_metrics(self):
+        # reset the count of every key to zero, 
+        # without removing it from Counter.elements()
+        for key in self._metrics_counter:
+            self._metrics_counter[key] = 0
```

Comment:
Kindof annoying that it has to be done this way, but it makes sense. If there were a way to do this without the member variable that would be a lot better. Especially since I'm not sure what will happen on server restarts. The game list will be cleared, but will the guage be reset properly?

Reply:
The `Gauge` object maintains all labels in a private dictionary `gauge._metrics`, see here:
https://github.com/prometheus/client_python/blob/2d85f4ae9ba6f16c87e40ce28e0b7111c9258c5a/prometheus_client/metrics.py#L110-L113

So if you are OK with depending on internals of the prometheus_client module, we can use that to get all labels used so far, instead of the `Counter`.

Another alternative is, of course, to only keep track of a fixed number of `game_mode`s (e.g. "faf", "ladder1v1", and "other").
The latter would also help with properly resetting at server restart.

Reply:
I guess we could use `GameService.featured_mods` which is the list of available featured mods from the database. This would be the same list used by the clients.